ynput/ayon-core
# Maya: add support for Maya 2025

## Changelog Description
Adding support to Maya 2025

## Additional info
This is adding default values to settings, moving around `capture.py` to Maya addon (where it should be anyway) and fixing some Qt imports there.

> [!NOTE]
> Touching `pyproject.toml` is done just to calm codespell in pre-commit hook because of the TVPaint path.

## Testing notes:
- Run Maya 2025 and it should work
- Run Maya 2024 should work too
- Playblasting should work also
